modules/services/agentless-scanning/variables.tf

```diff
@@ -26,3 +26,13 @@ variable "tags" {
     "product" = "sysdig-secure-for-cloud"
   }
 }
+variable "deploy_global_resources" {
+  description = "(Optional) Set this field to 'true' to deploy EventBridge to an AWS Organization (Or specific OUs)"
```

Comment:
Is this description as expected for agentless scanning feature?

Reply:
Yeah this is definitely not right. We should just substitute, `Eventbridge` with `Agentless Scanning`.
cc: @nkraemer-sysdig

Reply:
Done, fixed the description.